### PR TITLE
docs: modify documentation for `priority_rules` at top-level

### PR DESCRIPTION
**Pull request by Greesb**
Fixes MRGFY-3835
Depends-On: https://github.com/Mergifyio/engine/pull/13474

### CODE REVIEW

**Review by sileht — APPROVED** [review]

File: public/mergify-configuration-schema.json [hunk 1]
```diff
@@ -78,6 +78,12 @@
         "$ref": "#/definitions/QueueRule"
       }
     },
+    "priority_rules": {
+      "type": "array",
+      "items": {
+        "$ref": "#/definitions/PriorityRule"
+      }
```
Comment (jd) [inline public/mergify-configuration-schema.json:85]: Missing description?

**Review by jd — CHANGES_REQUESTED** [review]

File: public/mergify-configuration-schema.json (hunk 1 above)
Comment (Greesb) [reply public/mergify-configuration-schema.json:85]: No it's already defined, line 579, since it was used in the queue rules

Comment (jd) [reply public/mergify-configuration-schema.json:85]: For example, I don't see anywhere the sentence "The rule with the highest priority value wins." in the new code path?

Comment (jd) [reply public/mergify-configuration-schema.json:85]: I don't think that's a valid reason not to include the description. This file is going to be used by the dashboard for example to generate content.

Comment (Greesb) [reply public/mergify-configuration-schema.json:85]: I have no idea, i just copied what was already done. I'll add a description.

Comment (Greesb) [reply public/mergify-configuration-schema.json:86]: Done.

**Review by jd — DISMISSED** [review]

File: src/content/docs/merge-queue/priority.mdx [hunk 2]
```diff
@@ -308,6 +265,7 @@ If a pull request ends up in an unexpected position in the queue:
 - Make sure there isn't another priority rule that is placing the pull request
   at a different position.
 
-- Confirm that your priority rules are correctly set up in the queue rule. The
-  priority rules should be nested under the correct queue rule in your
-  configuration file.
+- Confirm that your priority rules are correctly set up at the top level of your
+  configuration file. Also make sure that there aren't priority rules defined
+  both at the top level and in the under the `queue_rule` section (which is now deprecated),
```
Comment (Syffe) [inline src/content/docs/merge-queue/priority.mdx:270]:
```suggestion
  both at the top level and under the `queue_rule` section (which is now deprecated),
```

**Review by Syffe — APPROVED** [review]

**Review by sileht — APPROVED** [review]

**Pull request merged**